src/libaktualizr/config/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set(HEADERS config.h)
+#set(HEADERS config.h)
```

Comment:
This (and the other commented-out line below) should be removed if they really aren't needed anymore.